can_all_node:
  ros__parameters:
    calibration_params: [1,34,1,0,0,0,1,0,1,0,0,1,1,1,0,0,0,0,0,0,0,0]
    can_interface: "can0"